src/data/aktivitetskrav/aktivitetskravTypes.ts

```diff
@@ -47,6 +47,13 @@ export interface AktivitetskravVurderingDTO {
   beskrivelse: string | undefined;
   arsaker: VurderingArsak[];
   frist: Date | undefined;
+  varsel: AktivitetskravVarselDTO | undefined;
+}
+
+export interface AktivitetskravVarselDTO {
+  uuid: string;
+  createdAt: Date;
+  svarFrist: Date;
```

Comment:
`svarfrist` i et ord? 😊
```suggestion
  svarfrist: Date;
```

Reply:
Ja, det er bedre. Får oppdatere DTOen i isaktivitetskrav først.